galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/compilation/simbank/AbstractCompilationLocalSimBank.java

```diff
@@ -57,6 +57,10 @@ private Path setupSimPlatform() throws ResourceUnavailableException, IOException
         refactorSimplatform(simplatformParent);
         
         outputFiles("simplatform-refactored", simplatformParent, false);
+
+         
+        makeGradleProperties();
```

Comment:
I think what this test is doing, is taking files that already exist in the Simplatform repository on Github and putting them in a new file structure for a test to use. As the gradle.properties file you are trying to create does not exist in Github, you will have to create it then allow it to be copied over like the other folders are.

- Remove this method and call a method on like 54 called 'createParentGradleProperties(simplatformParent)' and implement this method doing a similar thing to what createParentSettings does, but for your gradle.properties file.